src/index.js

```diff
@@ -54,411 +57,418 @@ var isSameAction = function(a, b) {
 
 var shouldRestart = function(a, b) {
   for (var k in merge(a, b)) {
-    if (a[k] === b[k] || isSameAction(a[k], b[k])) b[k] = a[k]
-    else return true
+    if (a[k] !== b[k] && !isSameAction(a[k], b[k])) return true
+    b[k] = a[k]
   }
 }
 
-var patchSub = function(sub, newSub, dispatch) {
-  for (var i = 0, a, b, out = []; i < sub.length || i < newSub.length; i++) {
-    a = sub[i]
-    out.push(
-      (b = newSub[i])
-        ? !a || b[0] !== a[0] || shouldRestart(b[1], a[1])
-          ? [b[0], b[1], b[0](b[1], dispatch), a && a[2]()]
-          : a
-        : a && a[2]()
+var patchSubs = function(oldSubs, newSubs, dispatch) {
```

Comment:
Is this code golfed enough? I can almost read it 😆

Reply:
@okwolf I just pushed to this branch like a few seconds ago. Are you sure you're looking at the right code? :)

Reply:
I was referring to the new version of `patchSubs` being more readable. Hopefully not at the cost of too many bytes 🙃

Reply:
@okwolf Which one is more readable?

Reply:
Most folks seem to think the nested ternaries are less readable. I don't think it's by much though 🤷‍♂